WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -82,6 +82,13 @@ public async Task StopAsync(Wallet wallet, CancellationToken cancellationToken)
 		await CommandChannel.Writer.WriteAsync(new StopCoinJoinCommand(wallet), cancellationToken).ConfigureAwait(false);
 	}
 
+	public bool TryGetWalletStatus(string walletName, [NotNullWhen(true)] out CoinJoinClientState? walletCoinjoinStatus)
+	{
+		var result = CoinJoinClientStates.TryGetValue(walletName, out var walletCoinjoinStateHolder);
+		walletCoinjoinStatus = walletCoinjoinStateHolder?.CoinJoinClientState;
```

Comment:
Here instead of using the returned boolean `result` to control the program flow you are using the nullable `walletCoinjoinStateHolder`. That's not okay.